examples/platform/silabs/SilabsDeviceDataProvider.cpp

```diff
@@ -25,7 +25,7 @@
 
 #ifdef SIWX917_USE_COMISSIONABLE_DATA
 #include "DeviceConfig.h"
-#include "silabs_utils.h"
+#include "siwx917_utils.h"
```

Comment:
I don't think this exists anymore. Probably need to rebase. Well, all the change in that file seems wrong.

Reply:
removed